chainer/link_hooks/weight_standardization.py

```diff
@@ -0,0 +1,67 @@
+import chainer.functions as F
+from chainer import link_hook
+
+
+class WeightStandardization(link_hook.LinkHook):
+    """Weight Standardization link hook implementation.
+
+    This hook standardizes a weight by *weight statistics*.
+
+    See: Siyuan Qiao et. al., `Weight Standardization
+    <https://arxiv.org/abs/1903.10520>`_
+
+    Args:
+    eps (int): Numerical stability in standard deviation calculation.
```

Comment:
```suggestion
    eps (float): Numerical stability in standard deviation calculation.
```

I know the SN hook link implementation has this typo and I'm fixing this now. 🙇